backend/index.ts

```diff
@@ -20,3 +21,7 @@ app.get("/", (req: Request, res: Response) => {
 app.use(router);
 //connect to Db If connect to DB success start Server Else console log the error
 connect(app, port);
+
+app.use(express.json());
+
+app.post('/api/todos', createTodo);
```

Comment:
we are using the app in routes folder to hookup all the end points to our server ,will fix that but for future reference